### DISCUSSION TITLE
Are liveness/readiness health check endpoints available?

### DISCUSSION THREAD

**Discussion by ViliusS** [fsouza/fake-gcs-server#1398] [Q&A] [answered]
What would be the proper way to health check if fake-gcs-server is running in a Kubernetes environment? For now I'm using /storage/v1/b endpoint, but this produces standard HTTP logs every few seconds. Is there any internal endpoint available to achieve the same goal?

**Comment by fsouza** [comment, marked as answer]
You can use `/_internal/healthcheck` as a readiness and liveness probe, but it'll still be included the access logs.

**Reply by ViliusS** [reply]
Thanks. Works as expected.

P.S. I wish documentation had these internal endpoints documented. Previously, I have spent a few hours trying to find why resume doesn't work. When I found that I need to set externalUrl I spend another half an our while I found `_internal/config` and tried to understand from the code how it works.
